rpc/lib/types/types.go

```diff
@@ -90,8 +90,8 @@ func NewRPCRequest(id jsonrpcid, method string, params json.RawMessage) RPCReque
 	}
 }
 
-func (req RPCRequest) String() string {
-	return fmt.Sprintf("[%s %s]", req.ID, req.Method)
+func (request RPCRequest) String() string {
```

Comment:
what's wrong with `req`?

Reply:
Nothing, I believe this and `response` were changed as part of a `sed` command.